docs/frontend_utilisation_guide.md

```diff
@@ -24,48 +24,91 @@ The Pixel Streaming Frontend UI contains all the functionality for UI components
 
 ---
 
-## Getting Started
-### Installing the Scalable Pixel Streaming Frontend   
+## Scalable Pixel Streaming Frontend Pieces
+### The Library
+The Library is the part of the Scalable Pixel Streaming Frontend that consumes the Epic Games Pixel Streaming Frontend and UI Frontend. It includes all of the custom signalling server logic that Scalable Pixel Streaming signalling servers require to work. The Library can be either obtained through [Github](https://github.com/ScalablePixelStreaming/Frontend) or [NPM](https://www.npmjs.com/package/@tensorworks/libspsfrontend). To make use of the Library it must be initialised via HTML and Javascript. The Example below is but one example of initialize the Library.  
```

Comment:
I think we can change the following line from the end of this paragraph:

> The Example below is but one example of initialize the Library.

to 

> The library is written in TypeScript but configured to export as a UMD module and can be consumed by plain JavaScript and most JavaScript frameworks.

I also recommend removing the random capitalisation of the word *Library* throughout this paragraph too.

Reply:
resolved in 10ff7e05bce6278cce6dc9d9b79203fa451dd870 and a1ac40e997464ba507195878f2a8b257f42f4d27